### ISSUE TITLE
add UI feedback when using yes/no keyboard shortcuts

### ISSUE THREAD

**Issue by Djfe** [common-voice/common-voice#1257]
shortcuts shouldn't only blink green/red when being clicked by cursors but also when shortcuts are used.  
the easiest solution might be to use the DOM and execute the click method on a button, which in return should also trigger the css, right?

**Comment by Gregoor** [comment]
Thanks for raising this, that feedback would indeed be valuable. Unfortunately it wouldn't be as simple as simulating a click as as that style is currently only applied on hover.

**Comment by Undigon** [comment]
Well, we can try using ```<input type="checkbox">``` and a bit of CSS+JS to implement onClic styling. We should then use some method to remove the checked property once the phrase has been updated. Is there any listener for that or should we apply a timed delay?

I've tried doing something like that before so I know it could be done (changing the colour "on checked" at least).

For reference, I got the idea some months ago from here: [Codepen example](https://codepen.io/oloman/pen/odnqy).

**Comment by zecakeh** [comment]
Also there should be visual feedback when you click/use the shortcut for YES but the sentence hasn't been played yet. So that people know it's because the full length of the audio file hasn't been played yet.

**Comment by moz-bozden** [comment]
Closing as stale...
